recipe/meta.yaml

```diff
@@ -22,8 +21,8 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://github.com/mlflow/mlflow/archive/refs/tags/v{{ version }}.zip
```

Comment:
@Jrice1317 Could you please add a comment why we change the source url? I think it's because pyproject.skinny.toml is missing, right?

Reply:
Correct!